### ISSUE TITLE
GPS testbench

### ISSUE THREAD

**Issue by gncemre23** [jks-prv/Beagle_SDR_GPS#571]
Do you have any testbench for the GPS part?

**Comment by jks-prv** [comment]
Do you mean a GNSS signal simulator? No I don't. You might look at Peter Monta's GNSS-DSP-tools project: https://github.com/pmonta/GNSS-DSP-tools

**Comment by gncemre23** [comment]
No, I mentioned the testbench for GPS hdl part like provided in Andrew Holme's project (GPS_tb.v).

**Comment by jks-prv** [comment]
No, I never used the simulation / testbench tools with ISE or Vivado during development.
